apps/dashboard/src/components/mutations/MutationsBalance.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+    <DataTable :rows="rows" :value="mutations" :rowsPerPageOptions="[5, 10, 25, 50, 100]" :paginator="paginator" lazy
+        @page="onPage($event)" :totalRecords="totalRecords" >
+        <Column field="moment" :header="$t('transactions.when')">
+            <template #body v-if="isLoading">
+                <Skeleton class="w-6 my-1 h-1rem surface-300" />
+            </template>
+            <template #body="mutation" v-else>
+                <span class="hidden sm:block">{{ mutation.data.moment.toLocaleDateString($i18n.locale, {
+                    dateStyle: 'full'
+                    }) }}</span>
+                <span class="sm:hidden">{{
+                    mutation.data.moment.toLocaleDateString('nl-NL', {
+                    dateStyle: 'short'
+                    })
+                    }}
+                </span>
+            </template>
+        </Column>
+
+        <Column field="createdBy" :header="$t('transactions.createdBy')">
+            <template #body v-if="isLoading">
+                <Skeleton class="w-6 my-1 h-1rem surface-300" />
+            </template>
+            <template #body="mutation" v-else>
+                {{ mutation.data.createdBy
+                && (mutation.data.createdBy?.id == userStore.current.user.id
+                    ? "You"
+                    : mutation.data.createdBy?.firstName + " " + mutation.data.createdBy?.lastName)
+                }}
+            </template>
+        </Column>
+
+        <Column field="mutationPOS" class="hidden sm:block" :header="$t('transactions.pos')">
+            <template #body v-if="isLoading">
+                <Skeleton class="w-6 my-1 h-1rem surface-300" />
+            </template>
+            <template #body="mutation" v-else>
+                {{ mutation.data.pos }}
+            </template>
+        </Column>
+
+        <Column field="change" :header="$t('transactions.amount')">
+            <template #body v-if="isLoading">
+                <Skeleton class="w-3 my-1 h-1rem surface-300" />
+            </template>
+            <template #body="mutation" v-else>
+                <div v-if="
+              mutation.data.type == FinancialMutationType.DEPOSIT ||
```

Comment:
(I) prefer to move logic and if statements to the script block, keeps the HTML cleaner.